pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -61,11 +61,11 @@ func (l *listUsersQuery) ListUsers(
 	foundUsersCh := make(chan *openfgav1.User, 1)
 	expandErrCh := make(chan error, 1)
 
-	foundUsers := make([]*openfgav1.User, 0)
+	foundUsersUnique := make(map[tuple.UserString]struct{}, 1000)
```

Comment:
Yes, for ListObjects it's the default `maxResults`:

https://github.com/openfga/openfga/blob/main/.config-schema.json#L58-L63

We don't have that flag for ListUsers yet so that is why I hardcoded it.